packages/web/src/common/store/notifications/fetchNotifications.ts

```diff
@@ -5,20 +5,23 @@ import {
   removeNullable
 } from '@audius/common'
 import { partition } from 'lodash'
-import moment from 'moment'
 import { call, fork } from 'typed-redux-saga'
 
 import { recordPlaylistUpdatesAnalytics } from './playlistUpdates'
 
 type FetchNotificationsParams = {
   limit: number
-  // ISO string
-  timeOffset: string
+  // unix timestamp
+  timeOffset?: number
   groupIdOffset?: string
 }
 
 export function* fetchNotifications(config: FetchNotificationsParams) {
-  const { limit, timeOffset, groupIdOffset } = config
+  const {
+    limit,
+    timeOffset = Math.round(new Date().getTime() / 1000),
```

Comment:
same here - may be worth putting a comment why we do this conversion

Reply:
for sure :)